src/components/dev-hub/icons/icons.stories.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, withKnobs } from '@storybook/addon-knobs';
+import ArrowheadIcon from './arrowhead-icon';
+import CopyIcon from './copy-icon';
+import EnvelopeIcon from './envelope-icon';
+import FacebookIcon from './facebook-icon';
+import FolderIcon from './folder';
+import GithubIcon from './github';
+import LinkIcon from './link-icon';
+import LinkedInIcon from './linkedin';
+import ListIcon from './list-icon';
+import LocationIcon from './location-icon';
+import MongoDBLeaf from './mdb-leaf';
+import MongoDBLogo from './mongodb-logo';
+import ShareIcon from './share-icon';
+import TwitchIcon from './twitch';
+import TwitterIcon from './twitter-icon';
+import Youtube from './youtube';
+import { colorMap, size } from '../theme';
+import styled from '@emotion/styled-base';
+
+const Row = styled('div')`
+    align-items: start;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: ${size.default} 0;
+    max-width: 80%;
+`;
+
+storiesOf('Icons', module)
+    .addDecorator(withKnobs)
+    .add('all', () => {
+        const color = select('color', Object.values(colorMap), colorMap.teal);
+        return (
+            <div>
```

Comment:
Can we maybe align the columns as well? Its sort of hard to see what's going on here!

Reply:
Yea that sounds good. Since Jordan has the Mdx stuff working I think I'll go ahead and close this out for the sake of his new PR because originally we wanted write our stories in an mdx format. From there I can implement this feedback.